tests/Unit/LocaleTest.php

```diff
@@ -27,6 +27,27 @@ public function testEnglishHungarianConsistency()
             $hu_keys = array_keys($hu_expressions);
             // Having the same keys in each file for both locales.
             $this->assertEquals($en_keys, $hu_keys);
+            // Having the same keys in validation.attributes for both locales.
+            if($file_in == 'validation.php') {
+                $this->assertEquals(array_keys($en_expressions['attributes']), array_keys($hu_expressions['attributes']));
+            }
+            // The key names do not contain any points.
+            foreach ($en_expressions as $key => $value) {
```

Comment:
This should be a different test (it's not the best practice to put everything in one test case, even if it's a bit faster that way).